tools/license-builder.sh

```diff
@@ -36,6 +36,8 @@ licenseText="$(tail -n +3 "${rootdir}"/deps/cares/LICENSE.md)"
 addlicense "c-ares" "deps/cares" "$licenseText"
 licenseText="$(cat "${rootdir}"/deps/cjs-module-lexer/LICENSE)"
 addlicense "cjs-module-lexer" "deps/cjs-module-lexer" "$licenseText"
+licenseText="$(cat "${rootdir}/deps/v8/third_party/ittapi/LICENSE")"
```

Comment:
```suggestion
licenseText="$(cat "${rootdir}/deps/v8/third_party/ittapi/LICENSES/BSD-3-Clause.txt")"
```

Reply:
Yes I found the wrong license position. Commited.